package.json

```diff
@@ -4,7 +4,10 @@
   "files": [
     "pkg-node/uplc_wasm_bg.wasm",
     "pkg-node/uplc_wasm.js",
-    "pkg-node/uplc_wasm.d.ts"
+    "pkg-node/uplc_wasm.d.ts",
+    "pkg-web/uplc_wasm_bg.wasm",
+    "pkg-web/uplc_wasm.js",
+    "pkg-web/uplc_wasm.d.ts"
   ],
   "main": "pkg-node/uplc_wasm.js",
   "types": "pkg-node/uplc_wasm.d.ts"
```

Comment:
If we're including web compilations, then we probably want to have some named exports here, with `node` being the default version.

Reply:
I'm ok with removing for now. It's not needed for me personally